tests/test_ddk.py

```diff
@@ -97,6 +98,11 @@ def test_K96(self):
         for p in testp.keys():
             adf = self.modelJ1713.d_phase_d_param(self.toasJ1713, delay, p)
 
+    def test_SINI_input(self):
```

Comment:
Can you add a test to see what happens when loading a par file?

Reply:
updated the tests. It passes locally. Waiting for the online test.